### PR TITLE
Update animations README

### PR DESCRIPTION
**Pull request by guidezpl**
Add links to the spec and codelab

### CODE REVIEW

File: packages/animations/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: animations
 description: Fancy pre-built animations that can easily be integrated into any Flutter application.
-version: 2.0.0
+version: 2.0.1
```
Comment (stuartmorgan) [inline packages/animations/pubspec.yaml:3]: This needs a corresponding CHANGELOG update.

File: packages/animations/README.md [hunk 2]
```diff
@@ -9,10 +9,13 @@ cd example/
 flutter run --release
 ```
 
-## Material motion for Flutter
+## Material Motion for Flutter
```
Comment (stuartmorgan) [inline packages/animations/README.md:12]: Neither of the pages you've linked to capitalize "motion"; why are you doing so throughout (mostly) this PR?

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/animations/README.md (hunk 2 above)
Comment (guidezpl) [reply packages/animations/README.md:12]: You're right, fixed

File: packages/animations/pubspec.yaml (hunk 1 above)
Comment (guidezpl) [reply packages/animations/pubspec.yaml:3]: Done.

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Pull request merged**